runtime/nonfb/FbtTranslations.js

```diff
@@ -49,6 +49,19 @@ const FbtTranslations = {
   registerTranslations(translations: TranslationDict): void {
     translatedFbts = translations;
   },
+
+  getRegisteredTranslations(): TranslationDict {
+    return translatedFbts;
+  },
+
+  mergeTranslations(newTranslations: TranslationDict): void {
+    Object.keys(newTranslations).forEach(locale => {
+      if (translatedFbts[locale] === undefined) {
+        translatedFbts[locale] = {};
+      }
+      Object.assign(translatedFbts[locale], newTranslations[locale]);
```

Comment:
minor nit:
```js
      translatedFbts[locale] = Object.assign(
        translatedFbts[locale] ?? {}, 
        newTranslations[locale],
      );
```
```

Reply:
That's unfortunately not possible because it would not be merging the translations into `translatedFbts` but rather into an empty unrelated object. Tests are covering this case, you can try it.

Reply:
But `Object.assign(...)` will *return* its result and we're setting the entry in my suggestion above?  Just tested in the imported diff — seems to work.

Reply:
🙈 You are right! I completely missed the part that you are suggesting to assign the value back via `translatedFbts[locale] = …`

My apologies. I updated the PR with your suggestion.